db/flush_job.h

```diff
@@ -164,6 +164,8 @@ class FlushJob {
   IOStatus io_status_;
 
   const std::shared_ptr<IOTracer> io_tracer_;
+
+  const std::string full_history_ts_low_;
```

Comment:
Just a question: why it's a `string` instead of `Slice`?

Reply:
A slice just points to some data that may or may not be owned. For flush job, it is better for the same FlushJob object to keep a copy of `full_history_ts_low_` so that it does not change throughout a flush. The cost of copy a small string is negligible compared to the cost of flush itself.